routers/api/packages/container/auth.go

```diff
@@ -27,11 +27,16 @@ func (a *Auth) Verify(req *http.Request, w http.ResponseWriter, store auth.DataS
 		return nil, err
 	}
 
-	if uid == 0 {
-		return nil, nil
-	}
-	if uid == -1 {
-		return user_model.NewGhostUser(), nil
+	if uid <= 0 {
+		switch uid {
+		case -1:
+			return user_model.NewGhostUser(), nil
+		case -2:
```

Comment:
```suggestion
		case user_model.ActionsUserID:
```

?

IIRC there is a const for it.

Reply:
I have a question, why `GhostUser` is not defined by consts.